database/model_config.go

```diff
@@ -1,10 +1,9 @@
 package database
 
 type DatabaseConfig struct {
-	MySql         *MySqlConfig
-	SqlLite       *SqlLiteConfig
-	DatabaseName  string
-	migrationsDir string
+	MySql        *MySqlConfig
+	SqlLite      *SqlLiteConfig
```

Comment:
This is called SQLite - https://sqlite.org/index.html 
I know it was misspelled before, but can we fix it?

Reply:
I'll change it!